shared-libs/bulk-docs-utils/src/bulk-docs-utils.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function(dependencies) {
+  dependencies = dependencies || {};
+  var Promise = dependencies.Promise;
+  var DB = dependencies.DB;
+
+  function getParent(doc) {
+    if (doc.type === 'person' && doc.parent && doc.parent._id) {
+      return DB.get(doc.parent._id);
+    }
+    return Promise.resolve();
+  }
+
+  return {
+    updateParentContacts: function updateParentContacts(docs) {
```

Comment:
It was my understanding that if you didn't give it an explicit function name then it would just show up as anonymous in the call stack, but after testing it in chrome this doesn't seem to be the case, so I'll remove!